summaries/3d_object_detection.md

```diff
@@ -0,0 +1,163 @@
+# Center-based 3D Object Detection and Tracking
+
```

Comment:
Mention authors and the conference

Reply:
can you see again i added the authors name and conference also two cents and resources. please check it once more

Reply:
Hi. Nice work. Just a small mistake, change CPVR to CVPR